ahk/templates/hotkeys.ahk

```diff
@@ -1,4 +1,8 @@
 #Persistent
+{% for directive in directives %}
+{% if directive.apply_to_hotkeys_process %}{{ directive }}{% endif %}
+{% endfor %}
```

Comment:
In case blocks get [trimmed](https://jinja.palletsprojects.com/en/3.0.x/templates/#whitespace-control), make the whitespace work in both configurations:
```suggestion
{% for directive in directives %}
{% if directive.apply_to_hotkeys_process %}

{{ directive }}
{% endif %}
{% endfor %}
```